### ISSUE TITLE
Use R8 API and impl.   New service wizard #5632 requires annotations, framework, scr, gogo shell

### ISSUE THREAD

**Issue by scottslewis** [bndtools/bndtools.workspace.min#7]

**Comment by scottslewis** [comment]
@pkriens  Please modify the bndtools.workspace.min workspace template so that

1) OSGi R8 is used
2) The new bndtools OSGi service wizard (in 7.0.0 as per https://github.com/bndtools/bnd/issues/5632 ) can use the repos provided to compile, resolve, and run the wizard-created projects + bndrun file.  

The requirements of the generated projects are currently:   Use of R8 org.osgi.annotation.bundle.Export and org.osgi.annotation.versioning.Version("1.0.0")  annotations.  Use of org.osgi.service.component.annotations.Component,  org.osgi.service.component.annotations.Reference, org.osgi.service.component.annotations.Activate scr annotations.    Use of R8 framework impl.   Use of R8 SCR impl, Use of gogo (shell, command, runtime bundles).

It might also make sense (IMHO) to add a R8 LogService API and impl, but the service wizard code does not currently use it.

**Comment by scottslewis** [comment]
I've now tested these changes with latest bndtools snapshot.   

Happy to report that the bndtools.workspace.min changes for R8 is now is now working as needed to allow the wizard-based creation of Java OSGi Services as per https://github.com/bndtools/bnd/issues/5632

Thanks to @chrisrueger for the leg-work on this.

I have one question for @pkriens :   The Java OSGi Service Wizard creates three new projects/bundles (api, impl, and consumer).   I've noticed that after these are created that the workspace needs a refresh to resolve bndrun deps (done on resolve).   I would like to programmatically trigger an entire workspace refresh *after* the projects are created by the wizard.   My question is:   How do I initiate a bndtools workspace refresh programmatically (I assume there is some service to call refresh on, but I don't know what it is).

Thanksinadvance.  Closing this issue.
